ap-microxrceddsgen/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Package: ap-microxrceddsgen
+Version: 0.0.1
```

Comment:
Ideally,this matches the version of the tool. Currently, it's not set, but official releases will have a version.

Reply:
yep, we can update this one, when you use the build script it update the Version to the right one's